chunky/src/java/se/llbit/chunky/renderer/DefaultRenderManager.java

```diff
@@ -398,6 +396,7 @@ private void updateRenderProgress() {
   protected void finalizeFrame(boolean force) {
     if (force || snapshotControl.saveSnapshot(bufferedScene, bufferedScene.spp)) {
       PostProcessingFilter filter = bufferedScene.getPostProcessingFilter();
+      if (mode == RenderMode.RENDERING) filter = PreviewFilter.INSTANCE;
```

Comment:
Does this mean that the filter will always be the preview filter?

Reply:
No this should be PREVIEW (as in `Scene.java`'s `postProcessFrame`). Will fix.